packages/x-components/tests/e2e/history-queries.feature

```diff
@@ -7,35 +7,25 @@ Feature: History queries component
     And   a popular searches API
     And   a related tags API
     And   a tracking API
-  @skip
-  Scenario Outline:  1. History query is clicked
+
+  Scenario Outline:  1. History query is clicked and deleted
     Given following config: hide if equals query <hideIfEqualsQuery>, debounce <debounceInMs>, requested items <maxItemsToStore>, rendered <maxItemsToRender>, instant search <instant>
     And   start button is clicked
     And   a "<list>" of queries already searched
     When  search-input is focused
     And   history query number <historyQueryItem> is clicked
     And   search-input is focused
-    And  related results are displayed
+    And   related results are displayed
     Then  the searched query is displayed in the search-box
     And   the searched query is not displayed in history queries if <hideIfEqualsQuery> is true
-
-    Examples:
-      | hideIfEqualsQuery | debounceInMs | maxItemsToStore | maxItemsToRender | instant | historyQueryItem | list                                                                                                             |
-      | true              | 150          | 15              | 8                | true    | 7                | puzzle, funko, puzzle 3d, funko pop, puzzle 3d harry, funko pop iron, puzzle 3d harry potter, funko pop iron man |
-      | false             | 150          | 15              | 8                | true    | 6                | puzzle, funko, puzzle 3d, funko pop, puzzle 3d harry, funko pop iron, puzzle 3d harry potter, funko pop iron man |
-
-  Scenario Outline: 2. History query delete button is clicked
-    Given following config: hide if equals query <hideIfEqualsQuery>, debounce <debounceInMs>, requested items <maxItemsToStore>, rendered <maxItemsToRender>, instant search <instant>
-    And   start button is clicked
-    And   a "<list>" of queries already searched
-    When  search-input is focused
+    When  clear search button is pressed
     And   the delete button of <historyQueryItem> is clicked
     Then  the deleted history query is removed from history queries
     And   the number of rendered history queries is <maxItemsToRender> - 1 if <maxItemsToStore> < <maxItemsToRender>
```

Comment:
IDE is complaining about this line (says that the step is not defined, but I can see it defined, maybe it's nothing
)

Reply:
It is defined on `history-queries.spec.ts`, line 81 onwards. For whatever reason it doesn't like `<` symbol... I can change it but we already have the same issue for the steps with a boolean parameter on them, so... let me know :)